packages.yml

```
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.8.0", "<0.9.0"]
+    version: [">=0.8.0", "<2.0.0"]
```

Does the lower bound need to be `1.0.0`? Or does `0.8.0` work for this package?
```suggestion
    version: [">=1.0.0", "<2.0.0"]
```

---

Good catch!